docs/setup/install/install-devtron-helm-2.md

```diff
@@ -57,7 +59,8 @@ Run following command
 kubectl -n devtroncd get installers installer-devtron -o jsonpath='{.status.sync.status}'
 ```
 
-Once installation process is complete, above command will print `Applied` It may take around 30 minutes for installation to complete.
+It may take around 30 minutes for installation to complete. You can use the above command to check the status of the installation, the installation is completed when it prints `Applied`. If the installation is still in progress, it will print `Downloaded`.
```

Comment:
Make it:
The install commands initiates Devtron-operator which spins up all the Devtron micro-services one by one in about 30 mins. You can use the above command to check the status of the installation if the installation is still in progress, it will print `Downloaded`. When the installation is complete, it prints `Applied`.

Reply:
Similarly change at other places too.